Control/lib/services/Bookkeeping.service.js

```diff
@@ -24,14 +24,15 @@ class BookkeepingService {
    * Constructor for configuring the service to retrieve data via Bookkeeping HTTP API
    * @param {Object} config = {url: string, token: string} - configuration for using BKP service
    */
-  constructor({url = '', token = ''}) {
-    this._url = url;
+  constructor({url, token, refreshRate}) {
+    this._url = url ?? '';
```

Comment:
Are you sure about the empty URL? It can probably lead to unexpected behavior. You should probably use null to be sure it's not provided